client/src/Visit.test.tsx

```diff
@@ -93,93 +129,72 @@ describe('Visit', () => {
     expect(genericErrors.length).toBe(1);
   });
 
-  it('renders a generic error when token throws an error', async () => {
-    const fetchTokenSpy = jest.spyOn(FetchToken, 'fetchToken');
-    fetchTokenSpy.mockImplementation(
-      async (): Promise<CommunicationUserToken> => {
-        throw new Error('test error');
-      }
-    );
-
+  it('should render JoinTeamsMeeting when config is loaded and meeting link is not set', async () => {
     const visit = mount(<Visit />);
 
     await runFakeTimers();
 
     visit.update();
 
     const spinners = visit.find(Spinner);
-    const genericErrors = visit.find(GenericError);
+    const joinMeetings = visit.find(JoinTeamsMeeting);
 
     expect(spinners.length).toBe(0);
-    expect(genericErrors.length).toBe(1);
+    expect(joinMeetings.length).toBe(1);
```

Comment:
Let's also find the TeamsMeeting and RoomsMeeting and assert that they are not rendered